src/borg/helpers/passphrase.py

```diff
@@ -111,19 +115,27 @@ def verification(cls, passphrase):
         ):
             print('Your passphrase (between double-quotes): "%s"' % passphrase, file=sys.stderr)
             print("Make sure the passphrase displayed above is exactly what you wanted.", file=sys.stderr)
+            print(
+                "Your passphrase (UTF-8 encoding in hex): %s" % bin_to_hex(passphrase.encode("utf-8")), file=sys.stderr
+            )
             try:
                 passphrase.encode("ascii")
             except UnicodeEncodeError:
-                print(
-                    "Your passphrase (UTF-8 encoding in hex): %s" % bin_to_hex(passphrase.encode("utf-8")),
-                    file=sys.stderr,
-                )
                 print(
                     "As you have a non-ASCII passphrase, it is recommended to keep the "
                     "UTF-8 encoding in hex together with the passphrase at a safe place.",
                     file=sys.stderr,
                 )
 
+    @staticmethod
+    def display_debug_info(passphrase):
```

Comment:
Maybe use a more structured output here, like:

Incorrect passphrase!

Passphrase used (without the quotes): "..."
Same, UTF-8 encoded, in hex: ...

Relevant Environment variables:
BORG_PASSPHRASE = "..."
...